pages/common/docker-save.md

```diff
@@ -0,0 +1,20 @@
+# docker save
+
+> Export one or more docker images to archive.
+> More information: <https://docs.docker.com/engine/reference/commandline/save/>.
+
+- Save the image by redirecting output to archive:
+
+`docker save {{image}} > {{path/to/file.tar}}`
```

Comment:
Based on the example 3, is this an `image:tag`?
```suggestion
`docker save {{image}}:{{tag}} > {{path/to/file.tar}}`
```

Reply:
yes